pyproject.toml

```diff
@@ -227,6 +235,9 @@ ignore = [
     "PLR09",   # Too many <...>
     "PLR2004", # Magic value used in comparison
     "ISC001",  # Conflicts with formatter
+    "EM101",   # raw-string-in-exception
```

Comment:
I would rather fix these, the justification given on the rule page seems reasonable to me: https://docs.astral.sh/ruff/rules/raw-string-in-exception/